zebra-consensus/src/primitives/halo2.rs

```diff
@@ -143,6 +144,15 @@ impl From<&zebra_chain::orchard::ShieldedData> for Item {
             .flags
             .contains(zebra_chain::orchard::Flags::ENABLE_OUTPUTS);
 
+        // FIXME: simplify the flags creation - make `Flags::from_parts` method pub?
+        // FIXME: support OrchardZSA?
+        let flags = match (enable_spend, enable_output) {
+            (false, false) => orchard::builder::BundleType::DISABLED.flags(),
+            (false, true) => orchard::bundle::Flags::SPENDS_DISABLED_WITHOUT_ZSA,
+            (true, false) => orchard::bundle::Flags::OUTPUTS_DISABLED,
+            (true, true) => orchard::bundle::Flags::ENABLED_WITHOUT_ZSA,
+        };
```

Comment:
The `Flags` type in `zebra_chain::orchard::shielded_data` needs a new associated constant like `const ENABLE_ZSA = 0b00000100` for its `ZcashSerialize`/`ZcashDeserialize` impls to parse the ZSA-enabled bit.

```suggestion
        // FIXME: support OrchardZSA?
        let flags = bundle::Flags::from_byte(shielded_data.flags.bits())
            .expect("type should not have unexpected bits");
```

Reply:
Link to this comment copied to https://github.com/QED-it/zebra/pull/37

Reply:
Fixed (added new ENABLE_ZSA flag and commited the suggestion).